index.html

```diff
@@ -169,74 +169,73 @@ <h2 id="about-me" class="aboutme_head">About me</h2>
         work and experience! If you like what you see and have a project you need coded, don’t hesitate to
         contact me.</p>
       <a class="web_button center_button" href="#">Get My Resume</a>
-    </div>
-    </div>
-    <div>
-      <ul class="flex_container_secondary">
-        <li class="nested_flexbox">
-          <h3>Languages</h3>
-          <img class="center" src="assets/images/icon-languages.svg" alt="Languages icon">
-          <ul>
-            <li>JavaScript</li>
-            <li>Ruby</li>
-            <li>HTML</li>
-            <li>CSS</li>
-          </ul>
-        </li>
-        <li class="nested_flexbox">
-          <h3>Frameworks</h3>
-          <img class="center ordering" src="assets/images/icon-frameworks.svg" alt="Frameworks icon">
-          <ul>
-            <li>Bootstrap</li>
-            <li>Ruby on rails</li>
-            <li>RSpec</li>
-            <li>Capybara</li>
-            <li>Selenium</li>
-          </ul>
-        </li>
-        <li class="nested_flexbox">
-          <h3>Skills</h3>
-          <img class="center" src="assets/images/icon-skills.svg" alt="Skills icon">
-          <ul>
-            <li>Codekit</li>
-            <li>GitHub</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-          </ul>
-        </li>
-      </ul>
-  </section>
-
-  <section>
-    <footer>
-      <h3 id="contact-info" class="contact_head">I'm always interested in hearing about new projects, so if you'd like
-        to chat please get in touch.</h3>
-      <form novalidate action="https://formspree.io/f/meqdykek" method="post">
-
-        <input type="text" id="full-name" maxlength="30" name="full_name" placeholder="First name" required>
-        <input type="email" id="email" name="user-email" placeholder="Email address"
-          pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" required />
-        <textarea name="message" maxlength="500" placeholder="Enter message here" required></textarea>
-        <div class="errorButton , flex">
-          <button class="form_button" type="submit">Get in touch</button>
-          <small class="collapse"></small>
-        </div>
-      </form>
-      <hr>
-      <ul class="footer_icon">
-        <li><a href="https://twitter.com/JoseADelphin"><img src="assets/images/social_media/twitter.svg"
-              alt="Twitter icon"></a></li>
-        <li><a href="https://www.linkedin.com/in/josedelphin/"><img src="assets/images/social_media/linkedin.svg"
-              alt="Linkedin icon"></a></li>
-        <li><a href="https://github.com/adelphinsucasa"><img src="assets/images/social_media/github.svg"
-              alt="Github icon"></a>
-        </li>
-      </ul>
-      <h4>dalejandro.005@gmail.com</h4>
-    </footer>
+      </div>
```

Comment:
- Kindly, use some background images as the Figma design in your about me section.👌



| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/206789547-9c4fb6f3-0682-4170-a253-a5fdacf70fba.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/206789553-578eb591-914e-45f3-87a8-eeefb42202b6.png" width="200">|


😎️